src/web/app/components/question-types/question-edit-details-form/text-question-edit-details-form.component.html

```diff
@@ -5,7 +5,7 @@
       Recommended length
     </span>
     for the response:
-    <input type="number" class="form-control" [ngModel]="model.recommendedLength" (ngModelChange)="triggerModelChange('recommendedLength', $event)" [disabled]="!isEditable" min="0">
+    <input type="number" class="form-control" [ngModel]="model.recommendedLength !== 0 ? model.recommendedLength : undefined" (ngModelChange)="triggerModelChange('recommendedLength', $event)" [disabled]="!isEditable" min="0">
```

Comment:
Have you manually verified that this works as intended? Just not sure whether returning `undefined` might provide the intended effect.

Reply:
Yup it works. But I will change it to use `''` instead for better clarity